### PR TITLE
LPS-73059 Limit bulk size to improve full reindex stability when documents are too large

### PR DESCRIPTION
**Pull request by arboliveira**

### CODE REVIEW

**Comment by arboliveira** [comment]
ci:test

**Comment by arboliveira** [comment]
ci:test

File: modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -1,3 +1,5 @@
+batch-indexing-bulk-sizes=Batch Indexing Bulk Sizes
+batch-indexing-bulk-sizes-help=The number of documents to index per batch, given a model type that supports bulk indexing. (e.g. com.liferay.journal.model.JournalArticle=10000) One entry per model. If not set, 10000 is used. Decreasing this value may improve full reindex stability when documents are too large.
```
Comment (rbohl) [inline modules/apps/portal-search/portal-search-api/src/main/resources/content/Language.properties]: @arboliveira can you explain this part to me? 

>Decreasing this value may improve full reindex stability when documents are too large.

**Comment by arboliveira** [comment]
ci:test

**Comment by BryanEngler** [comment]
sent to https://github.com/brianchandotcom/liferay-portal/pull/58717

**Pull request closed without merging**